spec/features/admin/banners_spec.rb

```diff
@@ -102,6 +102,16 @@
     expect(page).to have_link 'Such banner many text wow link', href: 'https://www.url.com'
   end
 
+  scenario "Update banner color when changing from color picker or text_field", :js do
```

Comment:
Also I would add a feature test to check if the banner created in admin is correctly shown in the web.